modules/dop/endpoints/projectpipelinefiletree.go

```diff
@@ -159,8 +159,14 @@ func (e *Endpoints) DeleteFileTreeNode(ctx context.Context, r *http.Request, var
 		return apierrors.ErrDeleteFileTreeNode.NotLogin().ToResp(), nil
 	}
 
+	inode, err := url.QueryUnescape(vars["inode"])
```

Comment:
what purpose must unescape inode?

Reply:
base64 stdEncoding params in url containing "/" result in 404, frontend will pass url encoding inode.